app/lib/clients/bamboohr_client.rb

```diff
@@ -0,0 +1,8 @@
+class Clients::BamboohrClient
+  def self.find_employees
+    client = Bamboozled.client(subdomain: AppEnv['BAMBOOHR_SUBDOMAIN'], api_key: AppEnv['BAMBOOHR_API_KEY'])
```

Comment:
make sure to update the env.sample and readme

Reply:
Currently we are working with free trial information and API keys that expire a week from setup. We are going to set up a new company that has an API key and has the information that we need for Mojo employees.  Should we wait until that is done before updating the readme? Otherwise I think we would need to update it every time we start a new trial. Would putting the information that I am currently using (API key and company domain) in this PR description be better?

Reply:
Adding placeholders to `.env.sample` is fine

```
BAMBOOHR_SUBDOMAIN=<ask a team mate>
```
or similar is fine. Sign posting requirements, and their purpose, in the readme is nice also.

Reply:
Okay thanks!